### ISSUE TITLE
Create requirements

### ISSUE THREAD

**Issue by torstein-vik** [torstein-vik/zophie#3]
Requirements is an essential part of a decent software development process, and must therefore be properly listed (in order of importance) before much more further planning can proceed.

**Comment by thea-vik** [comment]
I have created planning/REQUIREMENTS.md. @andreas1729, could you fill this file? No big rush, but it would be nice to have sooner rather than later.

**Comment by andreasholmstrom** [comment]
@torstein-vik I have created REQUIREMENTS-DETAILED.md in order to specify exactly how the content-addition should look in a first version. It is not complete, but the 1st level is finished. Could you have a look, maybe discuss it with me, and (when you have time) start implementing the 1st level?

**Comment by thea-vik** [comment]
@andreas1729 Sure, most of it seems well thought out, but perhaps some of the input fields should be merged? Furthermore, maybe the input fields should be organized in the GUI better than just a list of input fields? For now I will only implement 1 and then 6.

**Comment by andreasholmstrom** [comment]
Yes, perhaps they could be merged and then Zophie could figure out what to do which each input. If you think this is doable.
